src-tauri/src/platform_specific/linux.rs

```diff
@@ -1,8 +1,80 @@
 // Linux specific code
 
 use regex::Regex;
+use std::env;
 use std::process::Command;
 
+fn get_proton_dir() -> Option<String> {
+    let steam_dir = steamlocate::SteamDir::locate()?;
+    let compat_dir = format!("{}/compatibilitytools.d/", steam_dir.path.display());
+
+    Some(compat_dir)
+}
+
+/// Downloads and installs NS proton
+/// Assumes Steam install
+pub fn install_ns_proton() -> Result<(), thermite::prelude::ThermiteError> {
+    // Get latest NorthstarProton release
+    let latest = thermite::core::latest_release()?;
+
+    let temp_dir = env::temp_dir();
```

Comment:
I wouldn't namespace (or whatever it's called) a function if it's just called once

```suggestion
    let temp_dir = std::env::temp_dir();
```